pkg/bootstrap/bootstrap.go

```diff
@@ -266,3 +266,8 @@ func (b *Bootstrap) bootstrapEnabled(ctx context.Context) (bool, error) {
 	}
 	return true, nil
 }
+
+// Exported version for use in authz
+func (b *Bootstrap) BootstrapEnabled(ctx context.Context) (bool, error) {
+	return b.bootstrapEnabled(ctx)
+}
```

Comment:
If the function is needed in another package, I would rather just export the existing one rather than defining a new one here.